### PR TITLE
Add page (WIP, no merge yet)

### PR DESCRIPTION
**Pull request by bruce-fs**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@
 /tmp/
 
 *.gem
+.ruby-version
```
Comment (claudiofullscreen) [inline .gitignore]: Hello @bruce-fs -- This change looks unrelated to the current PR. Can you remove this? Thanks.

**Comment by bruce-fs** [comment]
push to diff repo

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ For more information about changelogs, check
 [Keep a Changelog](http://keepachangelog.com) and
 [Vandamme](http://tech-angels.github.io/vandamme).
 
+## 0.2.14 - 2017/01/24
+* [ENHANCEMENT] Add the Funky::Page API to fetch username and name
+  fields from the Facebook Graph API given a page ID
```
Comment (claudiofullscreen) [inline CHANGELOG.md]: @bruce-fs If new API is added to the Funky gem, it should be documented in README.md.

That is the place where currently all the public methods of Funky are described.

File: lib/funky/page.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  class Page
+    extend DataParser
+
+    attr_reader :data
+
+    def initialize(data)
+      @data = data
+    end
+
+    # @return [String] the video ID.
```
Comment (claudiofullscreen) [inline lib/funky/page.rb]: @bruce-fs You probably mean `page`, not `video`.

File: lib/funky/page.rb [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    def id
+      data[:id]
+    end
```
Comment (claudiofullscreen) [inline lib/funky/page.rb]: @bruce-fs It's good that you extracted the common part of `Video` and `Page` into an external file.

But as you can see, there are even more repetitions, like `def initialize` and `attr_reader :data` and `def id`.

To avoid all repetitions, you are better off having `Funky::Page` and `Funky::Video` both *inherit* from a base abstrat class, rather than extending them with a module.

File: .gitignore (hunk 1 above)
Comment (bruce-fs) [inline .gitignore]: updated in new pr

File: CHANGELOG.md (hunk 2 above)
Comment (bruce-fs) [inline CHANGELOG.md]: updated in new pr

File: lib/funky/page.rb (hunk 3 above)
Comment (bruce-fs) [inline lib/funky/page.rb]: updated in new pr

File: lib/funky/page.rb (hunk 4 above)
Comment (bruce-fs) [inline lib/funky/page.rb]: updated in new pr
